src/pages/{mdx.frontmatter__permalink}.js

```diff
@@ -37,4 +56,4 @@ export const query = graphql`
   }
 `
 
-export default BlogPost
\ No newline at end of file
+export default BlogPost
```

Comment:
Are you going to use default export or const export? This is a good moment to update the export in this component

Reply:
I know, but, the BlogPost component is not called in other sites, and if I change the export, I get an error in the RendererPager method, if I look for it, I can't find it.

![Captura desde 2022-12-08 11-27-41](https://user-images.githubusercontent.com/107500715/206471947-3bbfe730-04f7-4c58-a6ec-0746339560a0.png)
![Captura desde 2022-12-08 11-30-59](https://user-images.githubusercontent.com/107500715/206472759-ee979091-f2d1-4bc6-856b-ed9e05d33ece.png)
In the image above, the rest of the files in the browser are blogs, not components.

Reply:
My mistake